include/ignition/msgs/config.hh

```diff
@@ -15,4 +15,14 @@
  *
  */
 
+#ifndef IGNITION_MSGS__CONFIG_HH_
+#define IGNITION_MSGS__CONFIG_HH_
+
 #include <gz/msgs/config.hh>
+
+namespace [[deprecated("Use gz namespace.")]]  ignition
```

Comment:
Reminder to comment this out for now and suppress the `-Wattributes` warning

Reply:
No longer applicable since we're using pragma messages now https://github.com/gazebosim/gz-msgs/pull/252/commits/1c7fca05c6ad97565fdeb10b698d0802691d6328